src/Builder/CliMenuBuilder.php

```diff
@@ -117,8 +117,11 @@ public function addAsciiArt(string $art, string $position = AsciiArtItem::POSITI
      */
     public function addSplitItem() : SplitItemBuilder
     {
-        $this->menuItems[] = $id = uniqid('splititem-placeholder-', true);
-        
+        $this->menuItems[] = [
+            'type' => 'splititem-placeholder',
+            'id'   => $id = uniqid('', true),
```

Comment:
Should we allow optionnal user defined id ?
Do we have any interest in allowing to getSplitItem() like we do with getSubMenu() ?

Reply:
No - I don't want to add an API for something there is no concrete use case for. Same reason I'm in this mess in the first place.